src/osd/ECUtil.h

```diff
@@ -71,9 +71,6 @@ class stripe_info_t {
     return (offset / stripe_width) * chunk_size;
   }
   uint64_t chunk_aligned_logical_size_to_chunk_size(uint64_t len) const {
```

Comment:
This change means len no longer needs to be chunk aligned. I think you should refactor the function name to logical_size_to_chunk_size to reflect this.

Reply:
Not following here.  I think this is still used for its old purpose in the non-optimised code path.